test/compflags/link/sungeun/static_dynamic.skipif

```diff
@@ -35,6 +35,11 @@ elif $CHPL_HOME/util/printchplenv --make --all --internal \
   # a compopts-by-compopts basis, but we can't do that yet.
   #
   echo True
+elif [ ! -f "/usr/lib64/libpthread.a" ] ; then
```

Comment:
I think it should also check for /usr/lib/libpthread.a since not all platforms use lib64

Reply:
Hmm, yeah. And some platforms will use `/usr/lib/<target-triplet>/` and probably a multitude of places. I personally would be ok if this test is run on a few systems with this lib structure.

Alternatively I guess we could do something like:

```sh
find /usr/lib* -name libpthread.a 2>/dev/null
```

or maybe 

```sh
whereis libpthread | grep libpthread.a
```

But I'd be concerned about the time for `find` and am not sure if `whereis` is a good choice. I guess I'd either lean towards the current restricted check or something like `whereis`

Reply:
In a quick check none of the systems I regularly use have pthreads in `/usr/lib/`, but I do see an ubuntu system has pthreads in`/usr/lib/<target-triplet>/`.

Reply:
OK. Let's merge this as-in in that case.